apps/site/src/views/Home/components/Title/Title.tsx

```diff
@@ -0,0 +1,91 @@
+import { motion, useAnimate, stagger } from "framer-motion";
+import { useEffect } from "react";
+
+export default function Title() {
+	const [scope, animate] = useAnimate();
+
+	useEffect(() => {
+		animate(
+			"path",
+			{
+				pathLength: 1,
+				strokeLinecap: "round",
+			},
+			{
+				duration: 0.45, // time of letter animation
+				delay: stagger(0.2, { startDelay: 0.25 }), // speed of triggering letter animations
+			},
+		);
+	}, []);
+
+	return (
+		<svg
```

Comment:
Issue: The svg isn't responsive. We could set the width to be a percentage in the CSS.